.github/workflows/pr.yaml

```diff
@@ -58,7 +58,7 @@ jobs:
     steps:
       - name: Get PR Info
         id: get-pr-info
-        uses: rapidsai/shared-action-workflows/get-pr-info@branch-23.08
+        uses: nv-gha-runners/get-pr-info@main
```

Comment:
Also update `rapidsai/shared-workflows/.github/workflows/pr-builder.yaml@branch-24.02` to use `24.10`